main/lsp/test/error_reporter_test.cc

```diff
@@ -162,12 +192,129 @@ TEST_CASE("ErrorReporterIgnoresErrorsFromOldEpochs") {
         CHECK(output.empty());
     }
 }
+
+TEST_CASE("FirstAndLastLatencyReporting") {
+    auto gs = makeGS();
+    auto cs = makeConfig();
+    ErrorReporter er(cs);
+    {
+        core::UnfreezeFileTable fileTableAccess(*gs);
```

Comment:
We can address this in a cleanup PR later, but it's convention to re-freeze the file table once you're done with `enterFile`. e.g. on line 205 you can close the scope with `}` to re-freeze the file table.

I didn't mention it in previous PRs since it's nit-picky and doesn't influence the correctness of the tests here. :)